action.yml

```diff
@@ -1,27 +1,35 @@
-name: 'Setup Bitnami Kafka'
-description: 'Setup Bitnami Kafka'
-author: 'candis'
+name: "Setup Bitnami Kafka"
+description: "Setup Bitnami Kafka"
+author: "candis"
 inputs:
+  container_image_kafka:
```

Comment:
Shouldn't this be without underscores "_" to keep convention?

Reply:
Yup!